packages/core/spec/share/asciidoctor-spec.js

```diff
@@ -1665,6 +1665,117 @@ America/New_York
         expect(html).to.include('<style>pre.prism{background-color: lightgrey}</style>')
       })
     })
+    describe('Table stuff', function () {
+      it('should return true for hasRows()', function () {
+        const options = {}
+        const doc = asciidoctor.load('= Document Title\n\n|===\n\n|This is a title\n|===', options)
```

Comment:
Could you please use template literals? I find it easier to read.

Reply:
Done. Will be included in the next commit